dist/options/options.html

```diff
@@ -223,6 +226,21 @@ <h1>
 
             <textarea id="label-list" rows="16"></textarea>
 
+            <h2>
+                Date Grouping
+            </h2>
+
+            <div class="list-type-options">
+                <div class="list-type">
+                    <input type="radio" name="bundleDates" value="enable" id="enable-radio">
+                    <label for="enable-radio">Group messages based on labels and when they arrived</label>
+                </div>
+                <div class="list-type">
+                    <input type="radio" name="bundleDates" value="disable" id="disable-radio">
+                    <label for="disable-radio">Group messages based on labels only</label>
+                </div>
+            </div>            
```

Comment:
Instead of 2 radio buttons, I think this could be a checkbox, `Group messages by date`, since it's either enabled or disabled.

The other setting uses 2 radio buttons because it's less obvious what a checked vs unchecked state means.

Reply:
I've styled the checkbox as a toggle - leaving this open so you can review that specifically.

Preview: https://imgur.com/ZlEuFQr

Reply:
Beautiful!